colin/checks/labels/authoritative-source-url.py

```diff
@@ -0,0 +1,14 @@
+from colin.checks.abstract.labels import LabelCheck
+
+
+class AuthoritativeSourceUrlLabelCheck(LabelCheck):
```

Comment:
This is set automatically, based on the our document. I don't know if it make sense to have it in Fedora.

Reply:
I've added all such tests. Some of them are also in [general category](https://github.com/projectatomic/ContainerApplicationGenericLabels) so they are optional for Fedora.